package.json

```diff
@@ -62,12 +67,19 @@
     "@nrwl/nx-cloud": "11.1.3",
     "@nrwl/tao": "11.2.10",
     "@nrwl/workspace": "11.2.10",
+    "@nx-plus/docusaurus": "11.0.1",
+    "@tsconfig/docusaurus": "^1.0.2",
     "@types/jest": "26.0.8",
     "@types/node": "12.12.38",
+    "@types/react": "^17.0.9",
+    "@types/react-helmet": "^6.1.1",
+    "@types/react-router-dom": "^5.1.7",
```

Comment:
Hybrid project 😆

Reply:
Yes, Docusaurus builds on React and MDX and is developed by Facebook.